spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/RuntimeOperations.java

```diff
@@ -21,50 +21,71 @@
 import org.springframework.cloud.dataflow.rest.resource.AppStatusResource;
 import org.springframework.cloud.dataflow.rest.resource.StreamStatusResource;
 import org.springframework.hateoas.PagedModel;
+import org.springframework.http.HttpHeaders;
```

Comment:
@corneil did anything change in this file? I can't discern an actual change from the file format.